contracts/coordinator/src/client.rs

```diff
@@ -12,12 +12,20 @@ pub enum Error {
         "failed to execute ReadyToUnbond query at coordinator contract. verifier_address: {0}"
     )]
     ReadyToUnbond(String),
+    #[error(
+        "failed to execute VerifierDetailsWithProvers query at coordinator contract. service_name: {0}, verifier_address: {1}"
+    )]
+    VerifierDetailsWithProvers(String, String),
```

Comment:
prefer `VerifierDetailsWithProvers{service_name:String, verifier_address:String}`, it's not immediately obvious what the parameters mean and it's easy to get the order wrong since they have the same type